core/src/main/java/org/swblocks/decisiontree/domain/builders/DomainSerialiser.java

```diff
@@ -136,19 +142,31 @@ private static InputDriver getSingleDriver(final String currentDriver,
                 case REGEX:
                     inputDriver = new RegexDriver(currentDriver);
                     break;
+                case DATE_RANGE:
+                    final StringTokenizer tokenizer = new StringTokenizer(
+                            currentDriver.replace(DateRangeDriver.DR_PREFIX + ":",""), "|", false);
+                    if (tokenizer.countTokens() == 2) {
```

Comment:
EhSupport.ensure(tokenizer.countTokens() == 2, "DateRange driver incorrectly formatted") - does the case where the token count isn't 2 need to be handled?